packages/core/stdlib/math/_test.browse

```diff
@@ -0,0 +1,42 @@
+import "." as math
```

Comment:
Not on this PR, but we should write the 'assert' module and make this into an actual test.

Reply:
agreed

fwiw, I'm roughly copying deno's stdlib (https://deno.land/std@0.66.0) instead of node's

and they have a module called `testing` which exports assert and some other things